src/Microsoft.Identity.Web.TokenAcquisition/MergedOptions.cs

```diff
@@ -466,14 +474,14 @@ public void PrepareAuthorityInstanceForMsal()
             if (IsB2C && Instance.EndsWith("/tfp/", StringComparison.OrdinalIgnoreCase))

```

Comment:
Maybe a better fix could be to do it:
- here: https://github.com/AzureAD/microsoft-identity-web/blob/455c68776f44ec574738a0fc1b7b1f6039789e44/src/Microsoft.Identity.Web/WebApiExtensions/MicrosoftIdentityWebApiAuthenticationBuilderExtensions.cs#L183
- and here: https://github.com/AzureAD/microsoft-identity-web/blob/455c68776f44ec574738a0fc1b7b1f6039789e44/src/Microsoft.Identity.Web/WebAppExtensions/MicrosoftIdentityWebAppAuthenticationBuilderExtensions.cs#L325

Moving the code in AuthorityHelpers?

Reply:
That should also work.